### PR TITLE
Development

### PR DESCRIPTION
**Pull request by hannasoderstrom1**
Able to run tumor only.

New singularity image for sentieon hg38 with latest petasuite.

Added a panel of normal to use in tnscope for tumor only and the script that was used to create the PoN vcfs. 

Runs canvas as "normal only" for tumor only samples. Runs tnscope and manta as tumor only.

### CODE REVIEW

**Review by alvaralmstedt — COMMENTED** [review]
Would like more comments on all the if tumorid stuff that briefly explains why only parts of the pipelines are run in the to/no/tn cases

File: launch_snakemake.py [hunk 1]
```diff
@@ -82,8 +82,9 @@ def analysis_main(args, runnormal, output, normalname, normalfastqs, runtumor=Fa
 
         if output.endswith("/"):
             output = output[:-1]
-        if normalfastqs.endswith("/"):
-            normalfastqs = normalfastqs[:-1]
+        if normalfastqs:
+            if normalfastqs.endswith("/"):
```
[the remark this answers is not in the document]
Comment (hannasoderstrom1) [reply launch_snakemake.py:86]: I guess. This has always been there, not written by me :)

File: pipeline.snakefile [hunk 2]
```diff
@@ -67,20 +67,21 @@ fastq_dict["tumor"] = {}
...
-                fastqpair_pattern = os.path.basename(normal_fwd_fastq).replace(fwdpattern, "")
-                fastq_dict["normal"]["fastqpair_patterns"][fastqpair_pattern] = {}
-                fastq_dict["normal"]["fastqpair_patterns"][fastqpair_pattern]["fwd"] = normal_fwd_fastq
-    for revpattern in revpatterns:
-        normal_rev_fastqs = glob.glob(f"{normalfastqdir}/{normalname}*{revpattern}")
-        if normal_rev_fastqs:
-            for normal_rev_fastq in normal_rev_fastqs:
-                fastqpair_pattern = os.path.basename(normal_rev_fastq).replace(revpattern, "")
-                fastq_dict["normal"]["fastqpair_patterns"][fastqpair_pattern]["rev"] = normal_rev_fastq
+if normalfastqdirs:
```
[the remark this answers is not in the document]
Comment (hannasoderstrom1) [reply pipeline.snakefile:70]: yep

File: pipeline.snakefile [hunk 3]
```diff
@@ -105,15 +106,21 @@ if tumorfastqdirs:
 ###########################################################
 # Defining Non Cluster Rules
 if tumorid:
-    localrules: all, upload_to_iva, share_to_igv, tn_workflow, share_to_resultdir, excel_qc
+    if normalid:
```
[the remark this answers is not in the document]
Comment (hannasoderstrom1) [reply pipeline.snakefile:109]: solved in commit @ b124f2b

**Pull request merged**